index.html

```diff
@@ -279,7 +279,87 @@ <h2 class="about-me-heading">About me</h2>
           <li>Terminal</li>
         </ul>
       </div>
+
+      <!--CONTACTS-->
+      <div class="contact-info">
+        <p class="contact-text">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+
+        <!--contact form-->
+        <form action="https://formspree.io/f/xlezlyqd" method="POST">
+          <div class="form-group">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              class="input-style"
+              maxlength="30"
+              placeholder="Full name"
+              required
+            />
+          </div>
+
+          <div class="form-group">
+            <input
+              type="email"
+              name="email"
+              id="email"
+              class="input-style"
+              placeholder="Email address"
+              required
+            />
+          </div>
+
+          <div class="form-group">
+            <textarea
+              id="comments"
+              class="input-textarea"
+              name="comment"
+              maxlength="500"
+              placeholder="Enter message here"
+              required
+            ></textarea>
+          </div>
+
+          <a id="submit" class="submit-button" href="#">Get in touch</a>
```

Comment:
- [ ] As this is actually supposed to be a form `button` as opposed to a `hyperlink`. Please make sure you use the `button` or `input` tag for semantic purposes.
- [ ] At the moment, the `form` does not submit because of this, please kindly make sure the form submit to the formspree service

Reply:
Fixed. Changed to input tag at line #312 in index.html file